src/api/app/controllers/webui/session_controller.rb

```diff
@@ -8,36 +8,44 @@ def new; end
   def create
     user = User.find_with_credentials(params[:username], params[:password])
 
-    if user && !user.is_active?
-      redirect_to(root_path, error: 'Your account is disabled. Please contact the administrator for details.')
+    unless user.present?
```

Comment:
you don't need the `present?`, user is an object so that won't have any effect. :wink:

Reply:
Rubocop is failing also because of that, but removing the `present` should make Rubocop happy as well :joy:

Reply:
Ups! I forgot to run linters yesterday 😅